data/help.txt

```diff
@@ -65,9 +68,26 @@ help "lost 7"
 help "map"
 	`This map shows all the star systems you know about and the hyperspace links between them. You can use the map to plot a hyperspace course by clicking on the system you want to travel to. To initiate a jump to that system, close the map and press <Initiate hyperspace jump>.`
 
+help "map advanced"
+	`As you explore more of the galaxy, the star map will quickly become a very powerful tool. Clicking on a distant system will automatically plot the quickest route to it. If you have missions spread across several systems, you can shift-click each to plot a route between them, or control-click a system to inspect missions there without setting a route.`
+	`There are also many other useful modes the map can be set to. Click on one of the buttons in the lower right corner to learn more.`
+
+help "map advanced ports"
+	`The Ports panel provides detailed information about every planet, station, or other stellar objects you've come across. Most information can be clicked on (like the word "Outfitter", or on the price of a trade good), which will color every system on the star map appropriately.`
+	`For instance, you can click on "(has been visited)" to see which planets you've visited or not, or click on a trade good price to show how prices change throughout the galaxy. If you have a route planned already, this can be combined with a control-click to inspect trade good prices at stops along the way. A key will be shown on the right to let you know what colors mean in any given mode.`
+
+help "map advanced shops"
+	`Both the Shipyard and Outfitter panels give the same information: A catalog of any items you've seen in your career, providing detailed statistics at any time. Items will be in categories on the left side of the screen, which can be collapsed individually, or all at once with shift-click on a category name. You can also use the "Find" button in the bottom right to search by name.`
+	`When selecting an item, the details will appear in the top-right. Ports which sell it will be colored yellow, while ports with the correct shop but not the item will be colored teal. You can compare what you've selected with a second item with shift-click, which will put both details side-by-side.`
```

Comment:
```suggestion
	`The shipyard and outfitter map panels respectively display the ships and outfits sold on any planets that you have seen. Ships and outfits are displayed in categories which can be collapsed by clicking on the category name. Using shift+click on a category name will collapse or uncollapse all the categories at once. Each ship or outfit can be clicked on, displaying that item's stats while also highlighting any planets where it is sold. Using shift+click on a ship or outfit after you already have one selected will display that item's stats side by side the the previously selected item.`
```

Reply:
![image](https://user-images.githubusercontent.com/4471575/104132818-5216b200-5345-11eb-9011-46718041e407.png)
That ends up more succinct on paper, but in game it ends up being 18 lines without a line break of any kind. They are short lines, but it's still a big block of unbroken text. Could we just have a line break before "Each ship"?

Also, for <key>+click, that's what I started with, but then noticed `outfitter 2` uses "control-clicking". Do should we use + or - for those key combos? Or, are they separate and it's "<key>+click" and "<key>-clicking"?

Reply:
Outfitter 2 is a new help dialog as well. I'd standardize it to "+click," since it's describing two keys/actions being pressed at once, key 1 plus key 2. In the ship controls for example, we're using "control+number."

I also notice how in my suggestions I've lowercased "control," but in previous help dialogs it's capitalized "Control," as well as "Shift" and "Alt." I'd lowercase those, but since the older help dialogs use uppercase then I'd be find with just sticking with uppercase, so long as we remain consistent with it.